hoot-core/src/main/cpp/hoot/core/scoring/MapComparator.cpp

```diff
@@ -115,6 +115,8 @@ class CompareVisitor : public ElementConstOsmMapVisitor
     {
       in1.set("source:ingest:datetime","None");  // Wipe out the ingest datetime
       in2.set("source:ingest:datetime","None");
+      in1.set("source:datetime","None");  // Wipe out source datetime
+      in2.set("source:datetime","None");
```

Comment:
I assume that there is a unit test somewhere that ensures that datetimestamps are correctly pulled from the database since `MapComparator` won't be doing that anymore.  Is that correct?

Reply:
Hmmm. Maybe, maybe not. This was a quick fix for a failing test, because it looked like data without a source:datetime was getting defaulted to the "timestamp". I'll fix this in a more elegant way.

Reply:
Investigation reveals no internal consistency about this at all. In fact, the map-diff command-line argument that disables this chunk of code isn't even documented.

Reply:
My question is, if someone deletes, changes, screws-up the code that you fixed in the `OsmApiDbReader` class and the dates are screwed-up again, is there a unit test that will error out letting us know that `That 70's Problem` has reared its ugly head again?  I'd be willing to bet that if you ran this code as it sits without the `OsmApiDbReader` changes (just the changes to the unit tests) you'd get passing tests too.  There needs to be one unit test that explicitly checks for dates being correct.  If that is the case then all of the rest of the tests can ignore dates like this.